index.js

```diff
@@ -1,17 +1,53 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort(function(a,b){ 
```

Comment:
Boa! Daria pra fazer em uma linha somente

`this.items.sort((a, b) => a - b);`